en-US/ranking/criteria.md

```diff
@@ -46,70 +78,106 @@ The following requirements are for the individual maps themselves.
 * **The maps must be at least 45 seconds long.**
 * **The maps must be timed as accurately as possible.** Timing points must not be used in place of scroll velocity changes.
 * **The maps must have a song select preview point that sufficently compliments the song.** It's usually placed before the chorus or at the beginning of a section. A solid preview point attracts more players to try the map.
+* **BPM lines should not be used abusively for decorations or color changes.** This could be a confusing to players to play and redundant as in filesize.
+* **Objects must be snapped to timeline ticks according to snaps.** Use any custom snap as accurate as possible to the song.
 
 ### Difficulty Spread
 
 The following pertains to the spread of difficulty each mapset must follow.
+Rhythm-related rules and guidelines apply to approximately 160 BPM beatmaps. If your song is drastically faster or slower, some variables might be different, as detailed in Scaling BPM on the Ranking Criteria.
 
-##### Difficulty Names & Description
+##### Difficulty Names
 
 The standard names for difficulties in a mapset are Beginner, Easy, Normal, Hard, Insane, and Expert.
 Should you choose not to use the standard names, the difficulty names in your set must show clear sign of progression in difficulty consistent with that of the mapset and should be easy to interpret.
 
 The following descriptions are what each difficulty should typically look like. Keep in mind that this isn't concrete, and there is room for subjectivity in individual cases. It is only to serve a general outlook on how each map should look and feel.
 
+##### Difficulty Spread Rules
+
+| Length        | Minimum Required Maps |
+| ------------- | :-------------------: |
+| **0:45-2:29** |           2           |
+| **2:30+**     |           1           |
+
+##### Skipping Difficulties
+
+You are **not** permitted to skip difficulties for **sets with a song length between 0:45-2:29.** For instance, if you are making a 1 minute set with an Insane, you can either create a Hard or an Expert difficulty. Having a Beginner and an Insane is not allowed. **This rule does not apply to sets with a song length of 2:30 or longer.**
+
+##### Multi-Game Mode Sets
+
+Furthermore, **each game mode is treated separately.** If you are creating a mapset for both 4 Keys and 7 Keys, you must have a difficulty spread for both game modes.
+
+**If you have two game modes, you must preface each difficulty with either "4K" or "7K"** (Example: "7K Insane").
+
+## Guidelines
+
+The following are guidelines to consider when creating your mapset and submitting it for ranked status. **These are not mandatory and it can be broken under special circumstances.** Our intentions with this are to allow mappers to be creative with the types of maps they produce along with clear representation and not getting overmapped. As such, these guidelines are for bridges to increase your chances of getting your mapset ranked.
+
```

Comment:
I'm just gonna search for every mention of the word "must" here and replace it with "should" lol